src/Illuminate/Redis/Connections/PhpRedisConnection.php

```diff
@@ -155,6 +155,34 @@ public function lrem($key, $count, $value)
         return $this->command('lrem', [$key, $value, $count]);
     }
 
+    /**
+     * Removes and returns the first element of the list stored at key.
+     * It is the blocking version of LPOP.
+     *
+     * @param  dynamic  $arguments
+     * @return array|null
+     */
+    public function blpop(...$arguments)
```

Comment:
Could we change it to:
```PHP
public function blpop(...$arguments):?array
```

Reply:
Laravel does not ever use the language return types. Only phpdoc annotations.

Reply:
@GrahamCampbell. ok